continuous-integration/starter-kits/README.md

```diff
@@ -1,6 +1,177 @@
+you get started quickly with continuous integration best practices described in this overall guide. 
+Please see categories and links below for details. 
+
+## Continuous Integration with Compile, Release and Publish
+**Technologies:** Build integration using [Maven](https://maven.apache.org/), 
+[Github](https://github.com/), [Docker](https://www.docker.com/), [Jenkins](https://www.jenkins.io/) 
+and [Artifactory](https://jfrog.com/artifactory) for a [Java](https://openjdk.org/) application. 
+
+This section links to sample continuous integration projects for a Java application that 
+demonstrate how a build server, such as Jenkins, is used to verify that code compiles together 
+from different commits. It compiles projects in a pre-configured Docker container, and on success 
+publishes binaries to an artifact repository, such as Artifactory. Although this example focuses on 
+a Java-based project, it employs a generic [architectural paradigm](https://nasa-ammos.github.io/slim/continuous-integration/reference-architectures/) 
+that translates to builds in other languages. 
+
+**Prerequisites:** To leverage starter kits provided here, the following products may be 
+installed, tested and accessible locally: (Note that installation will benefit greatly from testing 
+and customization on local workstations.)
+* Administrative access to a **Github** repository that is the build source.
+* **Java** and **Maven** installed locally to test build conversions.
+* **Docker Desktop** to test container executions
+* Access to a **Jenkins** server, such as the enterprise CAE Jenkins at JPL.
+* Access to an artifact repository, such as the enterprise CAE Artifactory at JPL.
+
+For both examples described below, see the raw source which you can copy/paste into your own 
+project to configure a working build system.
+
+There are two configured examples: _(Maven setups supporting application designs)_
+1. `simple`: A monolithic compile of a single application that produces a single build artifact.
+2. `multimodule`: An multi-step build that compiles several modules and uses them as dependencies 
+in subsequent steps -- each step may produce its own artifacts, e.g. `JAR`s, `WAR`s or 
+other packaged build output. 
+
+### Requirements
+CI creates an ecosystem between products such that servers in compile, release and publishing 
+phases handoff processing between steps. All implementations are predicated on the following 
+configurations:
+1. Setup an account on a Jenkins server to host the build, such as the enterprise CAE Jenkins server 
+at JPL or another shared Jenkins host. 
+2. [Setup Webhook notifications](https://github.blog/2014-02-11-webhooks-level-up/) pointing to the 
+Jenkins instance with the URL of the server from the previous step that will host the build.
+3. Setup an account on an Artifactory server to host built binaries, such as the enterprise CAE 
+Artifactory server at JPL or another similar repository.
```

Comment:
Do we want to explicitly call out JPL's use of artifactory (and the "CAE" group it is under?)

Reply:
Yeah - making the language more institutionally-agnostic would be best.